recipes/process-assert-to-node-assert/src/workflow.ts

```diff
@@ -0,0 +1,230 @@
+
+		if (binding) {
+			// Handle member expressions like nodeAssert.strictEqual
+			replaceRules.push({
+				importNode: processImport,
+				binding,
+				rule: {
+					kind: 'member_expression',
+					has: {
+						kind: binding.includes('.') ? 'member_expression' : 'identifier',
+						pattern: binding,
+					},
+				},
+				replaceWith: 'assert',
+			});
+
+			// Handle standalone calls like nodeAssert(...)
+			replaceRules.push({
+				importNode: processImport,
+				binding,
+				rule: {
+					kind: 'call_expression',
+					has: {
+						kind: 'identifier',
+						field: 'function',
+						pattern: binding,
+					},
+				},
+				replaceWith: 'assert',
+			});
+		}
+
+		let hasNonAssertUsage = false;
+		for (const usage of processUsages) {
+			const propertyNode = usage.field('property');
+			if (propertyNode && propertyNode.text() !== 'assert') {
+				hasNonAssertUsage = true;
+				break;
+			}
+		}
+
+		if (!hasNonAssertUsage && processUsages.length > 0) {
+			processImportsToRemove.add(processImport);
+			linesToRemove.push(processImport.range());
+		}
+	}
+
+	const processedImports = new Set<SgNode<JS>>();
+
+	for (const replaceRule of replaceRules) {
+		const nodes = rootNode.findAll({
+			rule: replaceRule.rule,
+		});
+
+		for (const node of nodes) {
+			if (
+				replaceRule.importNode &&
+				!processedImports.has(replaceRule.importNode)
+			) {
+				if (!processImportsToRemove.has(replaceRule.importNode)) {
+					const removeBind = removeBinding(
+						replaceRule.importNode,
+						replaceRule.binding,
+					);
+
+					if (removeBind.edit) {
+						edits.push(removeBind.edit);
+					}
+
+					if (removeBind.lineToRemove) {
+						linesToRemove.push(removeBind.lineToRemove);
+					}
+				}
+				processedImports.add(replaceRule.importNode);
+			}
+
+			if (
+				replaceRule.rule.kind === 'member_expression' &&
+				replaceRule.binding
+			) {
+				// Replace the object part of member expressions (e.g., nodeAssert.strictEqual -> assert.strictEqual)
+				const objectNode = node.field('object');
+
+				if (objectNode) {
+					edits.push(objectNode.replace('assert'));
+				}
+			} else if (
+				replaceRule.rule.kind === 'call_expression' &&
+				replaceRule.binding
+			) {
+				// Replace the function identifier in call expressions (e.g., nodeAssert(...) -> assert(...))
+				const functionNode = node.field('function');
+
+				if (functionNode) {
+					edits.push(functionNode.replace('assert'));
+				}
+			} else {
+				const replaceText = replaceRule.replaceWith || 'assert';
+				edits.push(node.replace(replaceText));
+			}
+		}
+	}
+
+	let sourceCode = rootNode.commitEdits(edits);
+
+	sourceCode = removeLines(sourceCode, linesToRemove);
+
+	if (edits.length === 0 && linesToRemove) return sourceCode;
+
+	const alreadyRequiringAssert = getNodeRequireCalls(root, 'assert');
+	const alreadyImportingAssert = getNodeImportStatements(root, 'assert');
+
+	if (alreadyRequiringAssert.length || alreadyImportingAssert.length)
+		return sourceCode;
```

Comment:
```suggestion
	if (alreadyRequiringAssert.length || alreadyImportingAssert.length) return sourceCode;
```

Reply:
the auto format of vscode undo this change